### PR TITLE
remove Page and PageElement in favor of utility classes

### PR DESCRIPTION
**Pull request by flashdesignory**
Uses helper / utility functions to access elements.

### CODE REVIEW

File: packages/tools/workload-testing-utils/scripts/workload-testing-utils.js [hunk 1]
```diff
@@ -110,69 +104,21 @@ export function getParent(lookupStartNode, path) {
...
-        const e = this.document.elementFromPoint(
-            (rect.width / 2) | 0,
-            (rect.height / 2) | 0
-        );
-        return e;
-    }
+export function getElement(selector, path = [], lookupStartNode = document) {
+    const element = getParent(lookupStartNode, path).querySelector(selector);
+    return element;
+}
```
Comment (flashdesignory) [inline packages/tools/workload-testing-utils/scripts/workload-testing-utils.js:110]: replacement for Page.querySelector()

File: packages/tools/workload-testing-utils/scripts/workload-testing-utils.js [hunk 2]
```diff
@@ -110,69 +104,21 @@ export function getParent(lookupStartNode, path) {
...
-        const element = getParent(lookupStartNode, path).querySelector(selector);
-
-        if (element === null)
-            return null;
-        return element;
-    }
+export function getAllElements(selector, path = [], lookupStartNode = document) {
+    const elements = Array.from(getParent(lookupStartNode, path).querySelectorAll(selector));
+    return elements;
+}
```
Comment (flashdesignory) [inline packages/tools/workload-testing-utils/scripts/workload-testing-utils.js:115]: replacement for Page.querySelectorAll()

File: packages/tools/workload-testing-utils/scripts/workload-testing-utils.js (hunk 1 above)
Comment (camillobruni) [reply packages/tools/workload-testing-utils/scripts/workload-testing-utils.js:110]: contemplating whether this should be just querySelector or queryElement ... (anyway bokeshedding).

**Review by camillobruni — APPROVED** [review]
LGTM

File: packages/tools/workload-testing-utils/scripts/workload-testing-utils.js (hunk 1 above)
Comment (flashdesignory) [reply packages/tools/workload-testing-utils/scripts/workload-testing-utils.js:110]: we should create a twitter poll 🤣

**Pull request merged**